samples/NativeAOT/README.md

```diff
@@ -0,0 +1,60 @@
+# NativeAOT Sample
+
+This is the sample for using NativeAOT with .net Android
+
+## Debugging
+
+In order to debug we need to use `lldb`. The following commands will install the `lldb-server` for the application.
+
+```dotnetcli
+adb push $(ANDROID_NDK_HOME)/toolchains/llvm/prebuilt/darwin-x86_64/lib/clang/18/lib/linux/aarch64/lldb-server /data/local/tmp/lldb-server
+adb shell run-as net.dot.hellonativeaot cp /data/local/tmp/lldb-server .
+adb forward tcp:5039 tcp:5039
+adb shell run-as net.dot.hellonativeaot killall -9 lldb-server
+adb shell run-as net.dot.hellonativeaot ./lldb-server platform --listen "*:5039" --server
+```
+
+The above commands are setup in a MSBuild Target which will run after the `Install` target has run. So there
+is no need to do this manually if you are using `NativeAOT.csproj`.
+
+Once `lldb-server` is up and running, you will want to run up the app using the following
+
+```dotnetcli
+adb shell am start -S --user "0" -a "android.intent.action.MAIN" -c "android.intent.category.LAUNCHER" -n "net.dot.hellonativeaot/my.MainActivity" -D
+```
```

Comment:
If you remove the `-D`, do you need the `jdb` example below? Or is it needed for other reasons?

Reply:
The `-D` is there primarily to make sure the app stops on startup so you have time to attach the lldb debugger before the app starts. 

If you don't mind attaching after startup you can remove it.

Reply:
Ok, I think it makes sense to use it as a way to pause the app. 👍 

If an app is crashing on launch, you will need to connect the debugger before it starts.

Reply:
I'll add a comment that it is optional